iac-extensions/docker/src/test/java/org/sonar/iac/docker/utils/ArgumentUtilsTest.java

```diff
@@ -86,6 +86,7 @@ void shouldPartyVariableOrExpandableStrings(String input, String expectedOutput)
     "FROM foo\nARG foo=bar\nLABEL label=${foo:-notbar}",
     "FROM foo\nARG foo=notbar\nARG foo=bar\nLABEL label=$foo",
     "FROM foo\nARG foo=ar\nLABEL label=b$foo",
+    "FROM foo\nARG foo=barKey\nARG $foo=bar\nLABEL label=$barKey",
```

Comment:
Consider improving the formatting of the inputs. Right now it's hard to read it and see the difference. Maybe a space before each Instruction (and align in the same column)?

Reply:
Good point